#!/usr/bin/env python
PACKAGE = "llc"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("angularVelocity_P",	double_t,	0,	"Porptional",	0.666,	0,	100)
gen.add("angularVelocity_I",	double_t,	0,	"Porptional",	0,	0,	100)
gen.add("angularVelocity_D",	double_t,	0,	"Porptional",	 0,	0,	100)	

gen.add("linearVelocity_P",	double_t,	0,	"Porptional", 0.333,	0,	500)
gen.add("linearVelocity_I",	double_t,	0,	"Porptional", 0,	0,	100)
gen.add("linearVelocity_D",	double_t,	0,	"Porptional", 0,	0,	100)

exit(gen.generate(PACKAGE, "llc", "ControlParams"))
